src/views/Home.jsx

```diff
@@ -50,10 +53,9 @@ export function Home({ setListToken }) {
 						required
 					/>
 
-					<button type="submit">Submit</button>
+					<button type="submit">submit</button>
```

Comment:
I recommend changing `submit` to `Submit`. Generally, labels are written in sentence case (`I am a sentence`), and not lower case or `Title Case`.

Reply:
thanks for this Ej! It's updated now